index.html

```diff
@@ -1043,28 +1043,28 @@ <h2 class="mb-10"> Ready To <span> Download </span> </h2>
                     <div class="col-lg-8">
                         <ul class="links">
                             <li>
-                                <a href="home-app-landing.html" class="active">Home</a>
+                                <a href="" class="active">Home</a>

```

Comment:
For my homepage links, I like to set the href to `"/#"` just so it doesn't refresh the entire page.

Try giving the logo a click, then the clicking this link in the footer.

The other links are fine for now, since they are in progress.

Reply:
"/#" always trigger "can not find directory" I use just "#" instead, it has the same function.